tests/stubs/slimstat-tracker-stub.php

```
@@ -0,0 +1,20 @@
+namespace SlimStat\Tracker {
+    if (!class_exists(__NAMESPACE__ . '\Tracker')) {
+        class Tracker {
+            public static function slimtrack_ajax() { return '1'; }
+        }
+    }
+}
+
+namespace SlimStat\Services\Privacy {
+    if (!class_exists(__NAMESPACE__ . '\ConsentHandler')) {
+        class ConsentHandler {
+            public static function handleBannerConsent($json = true, $data = null) { return true; }
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$json'. (undefined)

(UnusedFormalParameter)